src/routes/mission/[mission]/edit/+page.svelte

```diff
@@ -85,6 +89,22 @@
 		applyAction(result);
 	}
 
+	async function saveCompletion(comp: ID<Completion>, index: number) {
+		const fData = new FormData();
+		fData.append('completion', JSON.stringify(comp));
+
+		const response = await fetch('?/editCompletion', {
+			method: 'POST',
+			body: fData
+		});
+		/** @type {import('@sveltejs/kit').ActionResult} */
+		const result = await response.json();
+
+		applyAction(result);
+		shownComp[comp.id] = false;
+		originalMission.completions[index] = JSON.parse(JSON.stringify(comp));
+	}
```

Comment:
It's not too important but just so you know you can use form actions and add code that runs before and after them. https://kit.svelte.dev/docs/form-actions#progressive-enhancement-customising-use-enhance